ME3Explorer/FaceFXAnimSetEditor/FaceFXAnimSetEditorControl.xaml.cs

```diff
@@ -66,7 +62,7 @@ public FaceFXAnimSetEditorControl() : base("FaceFXAnimSetEditor")
 
         public FaceFXAnimSet FaceFX;
 
-        public List<FaceFXLineEntry> Lines;
+        public ObservableCollectionExtended<FaceFXLineEntry> Lines { get; set; } = new ObservableCollectionExtended<FaceFXLineEntry>();
```

Comment:
this and Animations should be get-only properties, as they are never set after initialization (and doing so would break the binding)